cmd/root.go

```diff
@@ -17,6 +17,12 @@ configuration checks using Open Policy Agent (OPA).
 Preflight checks are bundled into Packages`,
 }
 
+func init() {
+	for _, command := range rootCmd.Commands() {
+		setFlagsFromEnv("PREFLIGHT_", command.PersistentFlags())
```

Comment:
I think this wants to be `.Flags()` instead, so that it picks up any local flags for that command.

Reply:
I am not sure about the difference, but it does not work with `Flags`